cypress/integration/homepage/home_page_latest_message_spec.js

```diff
@@ -1,44 +1,70 @@
 import { ImageDisplayValidator } from '../../Contentful/ImageDisplayValidator';
+import { RequestFilter } from '../../Analytics/RequestFilter';
+import { amplitude } from '../../fixtures/event_filters';
 import { MessageQueryManager } from 'crds-cypress-contentful';
 
-describe('Testing the Current Message on the Homepage:', function () {
+describe('Tests the Current Message on the Homepage', function () {
   let currentMessage;
-  let relativeAutoplayURL;
-  before(function () {
+  let requestFilter;
+
+  before(() => {
+    //Fetch Current Message
     const mqm = new MessageQueryManager();
     mqm.getSingleEntry(mqm.query.latestMessage).then(message => {
       currentMessage = message;
-      currentMessage.getURL().then(url => {
-        relativeAutoplayURL = url.autoplay.relative;
-      });
     });
 
+    //Setup capture for events
+    cy.server();
+    requestFilter = new RequestFilter(amplitude.isVideoStarted);
+    cy.route({
+      method: 'POST',
+      url: 'api.amplitude.com',
+      onResponse: (xhr) => {
+        requestFilter.keepMatch(xhr.request);
+      }
+    });
+
+    //Navigate
     cy.ignorePropertyUndefinedTypeError();
     cy.visit('/');
   });
 
-  it('Current Message title, description, and image should match Contentful', function () {
-    cy.get('[data-automation-id="message-title"]').as('title');
-    cy.get('@title').text().should('contain', currentMessage.title.text);
-    cy.get('@title').should('have.attr', 'href', relativeAutoplayURL);
+  it('Checks title, image, and "View latest now" button have correct link', () => {
+    currentMessage.getURL().then(url => {
+      const relativeAutoplayURL = url.autoplay.relative;
 
-    cy.get('[data-automation-id="message-description"]').as('description');
-    cy.get('@description').normalizedText().then(elementText =>{
-      expect(currentMessage.description.unformattedText).to.include(elementText);
+      cy.get('[data-automation-id="message-title"]').as('title');
+      cy.get('@title').text().should('contain', currentMessage.title.text);
+      cy.get('@title').should('have.attr', 'href', relativeAutoplayURL);
+
+      cy.get('[data-automation-id="message-video"]').as('videoImagelink');
+      cy.get('@videoImagelink').should('have.attr', 'href', relativeAutoplayURL);
+
+      cy.get('[data-automation-id="watch-message-button"]').as('watchMessageButton');
+      cy.get('@watchMessageButton').should('be.visible');
+      cy.get('@watchMessageButton').should('have.attr', 'href', relativeAutoplayURL);
     });
+  });
 
+  it('Checks card image and, if Bitmovin video, player exists and video autoplays', () => {
```

Comment:
Same deal here. I'm always in favor of smaller more granular tests if possible. If I remember correctly Cypress will use the same loaded page to check these things so you should be able to break these up